.travis.yml

```diff
@@ -0,0 +1,15 @@
+sudo: required
```

Comment:
Why should `sudo` be required?

Reply:
Docker is normally not available to a normal user unless it is added to the `docker` group, hence why TravisCI needs `sudo`.